### PR TITLE
Feature/js coding style

### PR DESCRIPTION
**Pull request by bbenno**

### CODE REVIEW

File: admin/js/settings.js [hunk 1]
```diff
@@ -6,12 +6,16 @@
...
 lcSelectInit();
 addEventListeners();
 
-/** Initialize LC-select field. */
+/**
+ * Initializes LC-select targeting the select field.
+ */
 function lcSelectInit() {
-	new lc_select(
+	lc_select(
```
Comment (zimbcode) [inline admin/js/settings.js:18]: Did you that? As I understood we need to to call the constructor here...

File: public/js/public.js [hunk 2]
```diff
@@ -59,12 +66,20 @@ const rotaractCalendarOptions  = {
 			}
 		}
 	}
-}
+};
 
-var calendar;
+let calendar;
```
Comment (zimbcode) [inline public/js/public.js:71]: Did you test this one? Not sure if this comes without trouble concerning the scope...

File: admin/js/settings.js (hunk 1 above)
Comment (bbenno) [reply admin/js/settings.js:18]: Change was motivated by <https://jshint.com/docs/options/#nonew>

File: public/js/public.js (hunk 2 above)
Comment (bbenno) [reply public/js/public.js:71]: Change is motivated by <https://jshint.com/docs/options/#varstmt>. I have not thought about scoping...

**Review by zimbcode — APPROVED** [review]

**Pull request merged**